news/2024/2024-04-09-skinning-contest-unchained.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "Skinning Contest: Unchained"
+date: 2024-04-09 15:00:00 +0000
+---
+
+Showcase your unshackled creativity to its fullest by participating in the newest skinning contest!
+
+![](/wiki/shared/news/2024-04-09-skinning-contest-unchained/banner.jpg)
+
+The skinship team has thrown all caution to the wind and unlocked the ultimate skinning contest theme.... which is that there isn't one. Yes, you heard that right!
```

Comment:
I would seek out ways to remove the latter line and add an emphasis that there are no theme restrictions for this iteration to the thumbnail text.

Reply:
76dd24902877a115a3ca529ed69c46a0d4cc8f65 should be addressed here